Makefile

```diff
@@ -88,7 +88,7 @@ lint-rust:
 gocd:
 	rm -rf ./gocd/generated-pipelines
 	mkdir -p ./gocd/generated-pipelines
-	cd ./gocd/templates && jb install
+	cd ./gocd/templates && jb install && jb update
```

Comment:
Why do you need to update right after `jb install` ?

Reply:
If someone deletes the `jsonnetfile.lock` then `jb install` is needed.

If `jsonnetfile.lock` exists but its out of date with `jsonnetfile.json`, i.e. a version change like this PR, then `jb update` is needed to update the lock file and download the new version.

It would be fine 99% of the time to remove the `jb install`, but there is no harm in having both in the name of making things easy for folks.